docs/debug/mpcie1_mpcie2_ethernet_card.md

```diff
@@ -0,0 +1,124 @@
+Problems with APU2 when using Mini PCI-Express Dual Gigabit Ethernet Controller Card
+====================================================================================
+
+## Problems
+
+1. When Ethernet Controller Card is placed in mPCIe1 slot (J14) placed extension cards 
+is recognized by OS but standard Ethernet output placed on APU2 board can't receive 
+correct IP adress. So it doesn't work correct. 
+
+2. When Ethernet Controller Card is placed in mPCIe2 slot (J13) 2 additional Ethernet 
+controller are not visible in OS. Standard Ethernet output placed on APU2 board 
+works well.
+
+> Device detection has been checked using `ip a`, `lspci` and `dmesg`. The results are 
+shown in the table below.
+
+```
+	+-----+-----------------------------+
+	| dev |          visibility         |
+	+-----+--------------+--------------+
+	| eth | mPCIe1 (J14) | mPCIe2 (J13) |
+	+-----+--------------+--------------+
+	|  0  |     yes      |     yes      |
+	|  1  |     yes      |     yes      |
+	|  2  |     yes      |     yes      |
+	|  3  |     yes      |     no       |
+	|  4  |     yes      |     no       |
+	+--------------------+--------------+
+```
+
+## Mini PCI-Express Dual Gigabit Ethernet Controller Card elements
+
+On a board are located PCI bridge [ASMedia Technology Inc. Device 1182](http://www.asmedia.com.tw/eng/e_show_products.php?cate_index=112&item=134) and 2 Ethernet 
+controllers [Realtek Semiconductor Co., Ltd. RTL8111/8168/8411](http://www.realtek.com/products/productsView.aspx?Langid=1&PNid=13&PFid=5&Level=5&Conn=4).
+
+## Booting lags
+
+When Ethernet Controller is placed in the mPCIe1 slot (J14) and booting is at network 
+devices setup time occurs long lag. In this case `eth0` device can not get correct IP. 
+After this next booting processes take same normal time. The reason may be multiple 
```

Comment:
@acihy this timing analysis would require precise engineering terms (you can measure stuff in minicom). Please avoid using unclear statements. Quality of this kind of reports is very important to us.